### PR TITLE
Add Travis test script

### PR DESCRIPTION
**Pull request by MetcalfeTom**
To solve [this issue](https://github.com/RasaHQ/rasa_core/issues/1427)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - /tmp/cached/
+python:
+  - '3.5'
+  - '3.6'
+install:
+  - pip install -r requirements.txt
+  - pip install https://github.com/explosion/spacy-models/releases/download/en_core_web_md-2.0.0/en_core_web_md-2.0.0.tar.gz --no-cache-dir > jnk
+  - python -m spacy link en_core_web_md en
+script:
+  - py.test test_nlu.py
```
Comment (wochinge) [inline .travis.yml]: * `py.test` should be added to a `dev-requirements.txt`
* newline add the end of the file

File: test_nlu.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from rasa_nlu.training_data import load_data
+from rasa_nlu.model import Trainer
+from rasa_nlu import config
+
+def test_train_and_persist():
+    training_data = load_data("data/nlu_data.md")
+    trainer = Trainer(config.load("nlu_config.yml"))
+    interpreter = trainer.train(training_data)
+    test_model_directory = trainer.persist("./models/nlu", fixed_model_name="test")
+    assert interpreter.parse('hello') is not None
```
Comment (wochinge) [inline test_nlu.py]: why not check for the actual entity?

File: requirements.txt [hunk 3]
```diff
@@ -19,7 +19,7 @@ coloredlogs==10.0
 flask==1.0.2
 flask_cors==3.0.4
 scikit-learn==0.19.1
-rasa_nlu~=0.13.0
+rasa_nlu
```
Comment (wochinge) [inline requirements.txt]: same as in the other pr. I am not sure whether we should remove the version annotation :-/

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+cache:
+  directories:
+  - $HOME/.cache/pip
+  - /tmp/cached/
+python:
+  - '3.5'
+  - '3.6'
+install:
+  - pip install -r requirements.txt
+  - pip install https://github.com/explosion/spacy-models/releases/download/en_core_web_md-2.0.0/en_core_web_md-2.0.0.tar.gz --no-cache-dir > jnk
```
Comment (wochinge) [inline .travis.yml]: Wouldn't `python -m spacy download en_core_web_md` be more robust?

File: requirements.txt [hunk 5]
```diff
@@ -31,3 +31,4 @@ pymongo==3.5.1
 python-dateutil==2.7.3
 spacy==2.0.12
 sklearn_crfsuite==0.3.6
+msgpack==0.5.6
```
Comment (wochinge) [inline requirements.txt]: Do we actually need this?

File: .travis.yml (hunk 4 above)
Comment (MetcalfeTom) [reply .travis.yml]: This is copied directly from the `rasa_nlu` travis script, so I assume there is a reason.  If it turns out to be a deprecated reason then we can rectify it for the main repo as well.

**Pull request merged**